namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/05-certificate.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: sentence-planning-preprod.service.justice.gov.uk
```

Comment:
if you have one zone, you can add this cert and ingress as `preprod.sentence-planning.service.justice.gov.uk